Convid19Dashboard/src/helper/firebase.js

```diff
@@ -0,0 +1,18 @@
+import firebase from "firebase/app"
+import "firebase/database"
+
+const firebaseConfig = {
+  apiKey: "AIzaSyDZxeXYx0kmWwsMH-TzS6UjWYZklqmpK7o",
```

Comment:
We should move this in the cloud or use Secret Manager API https://dev.to/googlecloud/using-secrets-in-google-cloud-functions-5aem

Reply:
I concur. I left it in at the time at this is a readonly key now.